admin/services/data-access/completed-check.data.service.js

```diff
@@ -62,14 +62,23 @@ completedCheckDataService.sqlFindOne = async (checkCode) => {
  * @return {Promise<Array>}
  */
 completedCheckDataService.sqlFindByIds = async (batchIds) => {
-  let select = `SELECT 
-       chk.*,
-       cr.payload,
-       cs.code,
-       cs.description
-      FROM [mtc_admin].[check] chk LEFT JOIN
-           [mtc_admin].[checkResult] cr ON (chk.id = cr.check_id) JOIN
-           [mtc_admin].[checkStatus] cs ON (chk.checkStatus_id = cs.id)`
+  let select = `
+  SELECT 
+  chk.*,
+  cr.payload,
+  cs.code,
+  cs.description,
+  prr.code restartCode,
+  (SELECT COUNT(id) FROM [mtc_admin].[pupilRestart] pr WHERE pr.check_id = chk.id) restartCount,
```

Comment:
Please remember that restarts can be deleted

Reply:
Good spot!